examples/testapp/src/pages/add-sub-account/index.page.tsx

```diff
@@ -11,9 +19,44 @@ import { PersonalSign } from './components/PersonalSign';
 import { SendCalls } from './components/SendCalls';
 import { SpendPermissions } from './components/SpendPermissions';
 
+type SignerType = 'cryptokey' | 'secp256k1';
+
 export default function SubAccounts() {
   const [sdk, setSDK] = useState<ReturnType<typeof createCoinbaseWalletSDK>>();
   const [subAccountAddress, setSubAccountAddress] = useState<string>();
+  const [signerType, setSignerType] = useState<SignerType>('cryptokey');
+  const [getSubAccountSigner, setGetSubAccountSigner] = useState<typeof getCryptoKeyAccount>(
+    () => getCryptoKeyAccount
+  );
+
+  useEffect(() => {
+    const stored = localStorage.getItem('signer-type');
+    if (stored !== null) {
+      setSignerType(stored as SignerType);
+    }
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem('signer-type', signerType);
+  }, [signerType]);
+
+  useEffect(() => {
+    const getSigner =
+      signerType === 'cryptokey'
+        ? getCryptoKeyAccount
+        : async () => {
+            let privateKey = localStorage.getItem('secp256k1-private-key') as `0x${string}` | null;
```

Comment:
nit: we should prefix this with `cbwsdk.demo.<key>` so its clear that the value is from the demo app in the sdk

Reply:
I also created a pk example for the import flow. Maybe we could reuse the pk across demos https://github.com/coinbase/coinbase-wallet-sdk/blob/master/examples/testapp/src/pages/import-sub-account/index.page.tsx#L38C35-L38C66

Reply:
This can be a follow up PR (either you or I can take)

Reply:
Happy to take that